wallet-new/src/Cardano/Wallet/API/WIP.hs

```diff
@@ -25,7 +25,7 @@ type API = Tags '["WIP"] :>
                    :> DeleteNoContent '[ValidJSON] NoContent
     :<|> "external-transactions" :> "unsigned"
                    :> Summary "Creates a new unsigned transaction (it will be signed externally)."
-                   :> ReqBody '[ValidJSON] PaymentWithChangeAddress
+                   :> ReqBody '[ValidJSON] Payment
```

Comment:
I am thinking that this is now useful as a standalone function regardless of external wallets. In the future we might want to come up with a satisfactory endpoint name and make this part of the "normal" API, as more than one exchange requested for the ability to have an unsigned transaction back (cc @KtorZ ).

Minor note: I tend to prefer the name `UnsignedTransaction` for consistency with our internal types, rather than `RawTransaction`.

Reply:
What he said ^^^ :+1: